.github/workflows/unit_tests.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: install dependencies
         run: |
           pip install -r requirements/development.txt
-          pip install nose
+          pip install pytest
```

Comment:
Today #484 was merged adding `nose` to `requirements/development.txt`. Could you change it there instead?

This also implies that the standalone `pip install pytest` line here is no longer needed.

Reply:
Sure, done.